internal/lsp/server.go

```diff
@@ -1887,6 +1896,17 @@ func (l *LanguageServer) loadWorkspaceContents(ctx context.Context, newOnly bool
 		return nil, fmt.Errorf("failed to walk workspace dir %q: %w", workspaceRootPath, err)
 	}
 
+	if l.bundleCache != nil {
+		rb, err := l.bundleCache.Refresh()
+		if err != nil {
+			return nil, fmt.Errorf("failed to refresh the bundle cache: %w", err)
+		}
+
+		if len(rb) > 0 {
+			fmt.Fprintln(os.Stderr, "Reloaded bundles:", rb)
```

Comment:
Do we need to print this each time?

Reply:
I've dropped it in 9d2aa3bfad3159476acf2b8ba9ddf0e47cddde9a. It'd be nice to have some log level control but that can come another time.